go/http/api.go

```diff
@@ -144,7 +144,17 @@ func (api *APIImpl) Check(w http.ResponseWriter, r *http.Request, ps httprouter.
 		remoteAddr = r.RemoteAddr
 		remoteAddr = strings.Split(remoteAddr, ":")[0]
 	}
-	checkResult := api.throttlerCheck.Check(appName, storeType, storeName, remoteAddr)
```

Comment:
This is a matter of preference, but given we're explicit separating the services in the API endpoint (`check`vs `check-read`) I'd rather have separate actions for the two services.